src/oic/oic/provider.py

```diff
@@ -1221,8 +1221,9 @@ def _do_user_info(self, token, **kwargs):
             logger.error('Wrong token type: {}'.format(typ))
             raise FailedAuthentication("Wrong type of token")
 
-        if _sdb.access_token.is_expired():
-            return error(error='invalid_token', descr='Token is expired', status_code=401)
+        if _sdb.access_token.is_expired(token):
+            return error(error='invalid_token', descr='Token is expired',
```

Comment:
Why are you changing this? Not that it matters but we have a line length set to 120 chars and this is certainly below that limit :)

Reply:
Right, just looked two lines down, where a similar check was wrapped.

Reply:
Yes, I noticed that as well. I think it is a difference between old code (which had a limit to 80) and new/changed code...

Reply:
Can revert if you care. Will look for the 120 thing in the future, my default is usually 80.

Reply:
No, I don't care enough for a revert.